stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/view/HiddenMessageListItemPredicate.kt

```diff
@@ -15,6 +15,6 @@ internal object HiddenMessageListItemFilter : MessageListView.MessageListItemFil
     }
 
     override fun predicate(item: MessageListItem): Boolean {
```

Comment:
Will the second part be executed for giphy messages? Looks like a short circuit operation which succeeds after execution of the first part

Reply:
Yeah, should be conjunction